openpathsampling/engines/trajectory.py

```diff
@@ -284,6 +284,28 @@ def next(self):
 
         return ObjectIterator()
 
+    def index_of(self, sub_traj):
+        """
+        Return a list of indices of the snapshots in sub_traj within the trajectory
+
+        Parameters
+        ----------
+        sub_traj : :obj:`Trajectory`
+
+        Returns
+        -------
+        list of int or None
+            a list of integer indices or None, if a snapshot is not found in the trajectory
+        """
+        ret = []
+        for s in sub_traj:
+            if s in self:
+                ret.append(self.index(s))
+            else:
+                ret.append(None)
+
+        return ret
+
```

Comment:
Sure, I forgot that you already did that. I would prefer that it can be used with a single trajectory and handle it correctly. Is that okay? I am sometimes annoyed when I need to wrap a single object in square brackets. This would require a `isinstance(traj, Trajectory)` though since both are iterable. Is it possible to merge #435 first? Or is this one required. Then I would copy your function to my PR. ...